publish/dir.targets

```diff
@@ -4,7 +4,7 @@
 
   <Target Name="GatherPublishToAzureBinaries">
     <PropertyGroup>
-      <RelativeBlobPath>$(Channel)/Binaries/$(SharedFrameworkNugetVersion)</RelativeBlobPath>
+      <RelativeBlobPath>Runtime/$(SharedFrameworkNugetVersion)</RelativeBlobPath>
```

Comment:
Just chatted with him -- I'm going to rebase my stuff once he merges his Checksum change in. It sounded like that change was very close to landing already.